src/Components/SingleArticleComments.jsx

```diff
@@ -133,8 +166,56 @@ export default function SingleArticleComments() {
         <br></br>
         -----------------------------------------------------------------{" "}
       </section>
-
-      {singleArticleComment.map((comment) => {
+      <section>
+        <h2>Delete a comment</h2>
+        <label> Login as: </label>
+        <br></br>
+        <select onChange={filterComments} required>
+          <option> </option>
+          {userNames.map((user) => {
+            return (
+              <option key={user} value={user}>
+                {" "}
+                {user}
+              </option>
+            );
+          })}
+        </select>
+        <br></br>
+        <br></br>
+        <label>Select comment (by ID) to delete:</label>
+        <br></br>
+        <select onChange={setDeleteComment}>
+          {" "}
+          <option> </option>
+          {placeholder.map((comment) => {
+            if (deleteUser !== comment.author) {
+              return (
+                <option disabled={true} key={comment.comment_id}>
+                  {" "}
+                  No Comments Made...{" "}
+                </option>
+              );
+            }
+            return (
+              <option
+                key={comment.comment_id}
+                value={comment.comment_id}
+                required
+              >
+                {" "}
+                {comment.comment_id}
+              </option>
+            );
+          })}
+        </select>
+        <br></br>
+        <br></br>
+        <button onClick={deleteComment}> Delete comment </button>
+        <br></br>
+        -----------------------------------------------------------------{" "}
+      </section>
+      {placeholder.map((comment) => {
```

Comment:
This does work, but I'm not a fan of the user experience! As a user of your site, I don't know the comment id of the comment I want to delete! This would be much nicer if you could conditionally add a delete button on the comments of a user that has been logged in